src/Groups/matrices/form_group.jl

```diff
@@ -640,3 +644,45 @@ function isometry_group(f::SesquilinearForm{T}) where T
       return G^Xf
    end
 end
+
+"""
+    isometry_group(L::AbsLat) -> MatrixGroup
+
+Return the group of isometries of the lattice `L`.
+
+The transformations are represented with respect to the ambient space of `L`.
+"""
+function isometry_group(L::Hecke.AbsLat)
+   # The caching needs Hecke >= 0.10.26
+   #z = get_special(L, :isometry_group)
+   #if z !== nothing
+   #  T = elem_type(base_field(L))
+   #  return z::MatrixGroup{T, dense_matrix_type(T)}
+   #end
+
+   if rank(L) == 0
+      # The following can be removed once we have Hecke >= 0.10.26
+      d = degree(L)
+      K = base_field(L)
+      gens = [identity_matrix(K,d)]
+   else
+      gens = Hecke.automorphism_group_generators(L)
+   end
+   G = matrix_group(gens)
+   # TODO: Cache the result.
+   # L._isometry_group = G
+   #set_special(L, :isometry_group => G)
+   return G
+end
+
+automorphism_group(L::Hecke.AbsLat) = isometry_group(L)
+
+aut(L::Hecke.AbsLat) = isometry_group(L)
```

Comment:
Do we really want this as an alias? Then we should do `const aut = automorphism_group` once, instead of defining this for some cases but not others.

However, these short names should only be added sparingly. Do we really want/need this?

Cc @fieker

Reply:
I agree, we should probably drop it. It is a bit too short. @simonbrandhorst: If this is fine for you, I would make the change to remove it.

Reply:
I have no strong feeling about this. I can just define the alias locally on my branch.